src/test/resources/com/puppycrawl/tools/checkstyle/xmllogger/ExpectedXMLLoggerErrorWithAndWithoutModuleId.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<checkstyle version="">
+<file name="InputXMLLoggerErrorWithAndWithoutModuleId.java">
+<error line="16" column="17" severity="error"
+               message="Name 'Method' must match pattern '^[a-z][a-zA-Z0-9]*$'."
+               source="com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck"/>
+<error line="16" column="17" severity="error"
+           message="Name 'Method' must match pattern '^[a-z][a-zA-Z0-9]*$'."
+           source="twCheck#com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck"/>
```

Comment:
should be `com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck#twCheck`